app/_how-tos/export-an-api-spec-as-a-document.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Export your API design document or request data
+content_type: how_to
+
+products:
+  - insomnia
+
+description: Export a design document, collection, or single request from Insomnia.
+
+tags:
+  - documents
+
+tldr:
+  q: How do I export an API design document, collection, or a request in Insomnia?
+  a: In your workspace, open the Document or Collection menu, click **Preferences → Data**, select the target format and scope, and then export the file.
+
+faqs:
+  - q: Can I export just one request?
+    a: Yes. Click **Export Data → Single Request** from the Document or Collection menu or click  **Preferences → Data**. Then select just that request.
+prereqs:
+  inline:
+    - title: API specification
+      content: |
+        You need to have an API specification in one of the following formats:
+          * Insomnia
+          * Postman v2
+          * HAR
+          * OpenAPI (versions 3.0, 3.1)
+          * Swagger
+          * WSDL
+          * cURL
```

Comment:
```suggestion
    - title: An Insomnia design document
      content: |
        You need to have an Insomnia design document. For instructions on creating one see [Create a design document](/how-to/create-a-design-document/)
```

Reply:
Since the doc title is: "Create a design document" you only need a design document. I tested and it seems like you don't need anything in that design document to export it.

Reply:
The original tl;dr you had was more relevant for importing, but it looks like once the information is imported it becomes either a document or a request collection.